DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py

```diff
@@ -666,7 +666,8 @@
 
     # append the approximate clusters monitoring for the HI run case
     from DQM.SiStripMonitorApproximateCluster.SiStripMonitorApproximateCluster_cfi import SiStripMonitorApproximateCluster
-    process.siStripApproximateClusterComparator = SiStripMonitorApproximateCluster.clone(compareClusters = cms.bool(True))
+    process.siStripApproximateClusterComparator = SiStripMonitorApproximateCluster.clone(compareClusters = cms.bool(True),
+                                                                                         ClustersProducer = cms.InputTag("hltSiStripClusterizerForRawPrime"))
```

Comment:
```suggestion
    process.siStripApproximateClusterComparator = SiStripMonitorApproximateCluster.clone(compareClusters = True,
                                                                                         ClustersProducer = "hltSiStripClusterizerForRawPrime")
```

Since you are there, you could fo the same also for `process.multFilter` slightly here above

Reply:
next PR.